components/esp_https_server/include/esp_https_server.h

```diff
@@ -99,6 +99,8 @@ struct httpd_ssl_config {
 
     void *ssl_userdata; /*!< user data to add to the ssl context  */
     esp_tls_handshake_callback cert_select_cb; /*!< Certificate selection callback to use */
+
+    const char** alpn_protos; /*!< Application protocols the server supports in order of prefernece. Used for negotiating during the TLS handshake, first one the client supports is selected. The data structure must live as long as the https server itself! */
```

Comment:
Please drop an initialization line for this member in `HTTPD_SSL_CONFIG_DEFAULT` (at the very end, after `cert_select_cb`)

Reply:
Done :+1:.

Maybe the CI should be adapted to catch this issue?